go.mod

```diff
@@ -10,11 +10,11 @@ require (
 	github.com/hashicorp/golang-lru v0.5.4
 	github.com/palantir/go-baseapp v0.2.4
 	github.com/pkg/errors v0.9.1
-	github.com/rcrowley/go-metrics v0.0.0-20200313005456-10cdbea86bc0
-	github.com/rs/zerolog v1.18.0
-	github.com/shurcooL/githubv4 v0.0.0-20191127044304-8f68eb5628d0
+	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
+	github.com/rs/zerolog v1.24.0
+	github.com/shurcooL/githubv4 v0.0.0-20210725200734-83ba7b4c9228
```

Comment:
Hm, this contains a lot of changes. How many of them are going to be breaks?https://docs.github.com/en/graphql/overview/changelog

Reply:
Because GraphQL queries are defined by applications, I think any applications impacted by these breaks would have already encountered them. The `githubv4` library's only coupling with the API is the helper types it defines, so some of those could have been removed or changed, but I'd expect queries using them are already broken.